docs/Compatibility.md

```diff
@@ -4,156 +4,259 @@ The Ruby Datadog Trace library is open source. See the [dd-trace-rb][1] GitHub r
-|       |                            | 2.7     | Full                                 | Latest              |
-|       |                            | 2.6     | Full                                 | Latest              |
-|       |                            | 2.5     | Full                                 | Latest              |
-|       |                            | 2.4     | EOL                                  | < 2.0.0             | [comment]: <> (# DEV-2.0: Update with release date)
-|       |                            | 2.3     | EOL                                  | < 2.0.0             |
-|       |                            | 2.2     | EOL                                  | < 2.0.0             |
-|       |                            | 2.1     | EOL                                  | < 2.0.0             |
-|       |                            | 2.0     | EOL since June 7th, 2021             | < 0.50.0            |
-|       |                            | 1.9.3   | EOL since August 6th, 2020           | < 0.27.0            |
-|       |                            | 1.9.1   | EOL since August 6th, 2020           | < 0.27.0            |
-| JRuby | https://www.jruby.org      | 9.3     | Full                                 | Latest              |
-|       |                            | 9.2     | Full                                 | Latest              |
+| Type  | Documentation              | Version   | Support type        | Gem version support |
+|-------|----------------------------|-----------|---------------------|---------------------|
+| MRI   | https://www.ruby-lang.org/ | 3.3       | [GA](#support-ga)   | Latest              |
+|       |                            | 3.2       | [GA](#support-ga)   | Latest              |
+|       |                            | 3.1       | [GA](#support-ga)   | Latest              |
+|       |                            | 3.0       | [GA](#support-ga)   | Latest              |
+|       |                            | 2.7       | [GA](#support-ga)   | Latest              |
+|       |                            | 2.6       | [GA](#support-ga)   | Latest              |
+|       |                            | 2.5       | [GA](#support-ga)   | Latest              |
+|       |                            | 2.4       | [EOL](#support-eol) | < 2.0.0             |
+|       |                            | 2.3       | [EOL](#support-eol) | < 2.0.0             |
+|       |                            | 2.2       | [EOL](#support-eol) | < 2.0.0             |
+|       |                            | 2.1       | [EOL](#support-eol) | < 2.0.0             |
+|       |                            | 2.0       | [EOL](#support-eol) | < 0.50.0            |
+|       |                            | 1.9.3     | [EOL](#support-eol) | < 0.27.0            |
+|       |                            | 1.9.1     | [EOL](#support-eol) | < 0.27.0            |
```

Comment:
Minor: Let's merge these together, it doesn't seem very relevant to have them separate anymore?

```suggestion
|       |                            | 1.9       | [EOL](#support-eol) | < 0.27.0            |```